announcements/1.12.5.md

```diff
@@ -0,0 +1,23 @@
+### Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.12.5.
+
+Please see [component versions](https://anbox-cloud.io/docs/component-versions) for a list of updated components.
+
+#### New features & improvements
+
+n/a
+
+#### Bugs
+
+* AC-676 Launching containers on a specific node fails with "node not found"
+* AC-671 Application that was signed with a custom system image  can't access hidden APIs
+* AC-667 Multi touch is not working when streaming from webrtc platform
+* AC-663 Latest firefox doesn't render in 1.12.x
+* AC-662 Android containers fails to start on Linux 5.13.x
```

Comment:
```suggestion
* AC-663 Latest Firefox doesn't render in 1.12.x
* AC-662 Android containers fail to start on Linux 5.13.x
```

Reply:
Usually I don't suggest fixes for the bug descriptions, but I feel these must be fixed.